domain-server/resources/web/wizard/js/wizard.js

```diff
@@ -171,6 +172,34 @@ function setupWizardSteps() {
   $(currentStep).show();
 }
 
+function promptToCreateDomainID() {
+    setTimeout(function(){ 
+        createDomainIDPrompt(function(label) {
+            var domainJSON = {
+              "label": label
+            }
```

Comment:
Inconsistent indenting.

Reply:
I'm going to use Atom to indent all these files to 4 lines after this, I feel like that'll be a better solution so no one has to deal with that again? :)

Reply:
There are pros and cons to reworking all the indenting. A con is that you lose git blame history for each line.

Reply:
Ah, darn it. Then I'm not sure anymore, it drives me mildly crazy though every time I go in there to fix something. Hmph. I'll fix it here then.